src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/DisputeCount.java

```diff
@@ -24,47 +28,57 @@
 @Getter
 @Setter
 @NoArgsConstructor
-public class DisputedCount extends Auditable<String> {
-
+public class DisputeCount extends Auditable<String> {
+	
+	/**
+	 * Primary key
+	 */
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
 	@Id
 	@GeneratedValue
-    private Long id;
-
-	/**
-	 * Represents the disputant plea on count.
-	 */
-	@Column
-	private Plea plea;
-
+	private Long disputeCountId;
+	
 	/**
 	 * The count number.
 	 */
 	@Column
 	@Min(1) @Max(3)
-	private int count;
+	private int countNo;
+	
+	/**
+	 * Represents the disputant plea on count.
+	 */
+	@Column(length = 3)
+	@Schema(nullable = false)
+	@Enumerated(EnumType.STRING)
+	private Plea pleaCode;
 
 	/**
 	 * The disputant is requesting time to pay the ticketed amount.
 	 */
 	@Column
-	private boolean requestTimeToPay;
-
+	@Schema(nullable = false)
+	@Enumerated(EnumType.STRING)
+    private YesNo requestTimeToPay;
```

Comment:
I'm just thinking that it might have been a cleaner solution to have hibernate map what boolean mean when it reads/write it to the database.  Ie. We can tell Hibernate to serialize a boolean as "Y" or "N" characters.  This way from the application point of view we only have to deal with booleans and leave the serialization to the database layer only.

https://thorben-janssen.com/hibernate-tips-how-to-map-a-boolean-to-y-n/